### PR TITLE
[ASM] Adding Slack Messages

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-10042

## Description
Adding Slack message to CASM playbook

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@capanw / @BigEasyJ , please review and I can demo

@Shellyber / @ssokolovich , please let me know if I need to do anything with the validation errors (I assume it isn't my issue)

**Comment by ShirleyDenkberg** [comment]
@Shellyber @ssokolovich Doc review completed.

**Comment by Shellyber** [comment]
@johnnywilkes Please update your branch with the updated master. You can use the script [git_pull_master_into_fork.sh](https://github.com/demisto/content/blob/master/contribution/utils/git_pull_master_into_fork.sh) to do so,
Also, you will need to fix the validation errors.

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_41.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+#### Playbooks
+
+##### Cortex ASM - ASM Alert
+
+Added support for Slack messaging option.
+
+##### New: Cortex ASM - Instant Message
+
+New: This playbook is used to create instant messages toward service owners to notify them of their internet exposures.<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
```
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_7_41.md]: [add XSIAM ver](https://github.com/demisto/content/pull/34827/commits/7ef5c57b5493349633aa13f13bf1104539020566)

File: Packs/CortexAttackSurfaceManagement/README.md [hunk 2]
```diff
@@ -184,6 +185,12 @@ A playbook that given the IP address enriches GCP information relevant to ASM al
 
 ![Cortex ASM - GCP Enrichment](https://github.com/demisto/content/raw/234817ee736286eee1b40ece897cb7b2974771dc/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_GCP_Enrichment.png)
 
+#### Cortex ASM - Instant Message
+
+A playbook that is used to create instant messages toward service owners to notify them of their internet exposures.
+
+![ortex ASM - Instant Message](https://raw.githubusercontent.com/demisto/content/f00e2443782383c76dcfa7e537c03940afe3197c/Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Instant_Message.png)
```
Comment (capanw) [inline Packs/CortexAttackSurfaceManagement/README.md:192]: I think C is missing here

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 360
+        }
+      }
+  "10":
+    continueonerrortype: ""
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml:143]: What should happen if the slack message fails to send?

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+      channel_id:
+        simple: ${inputs.InstantMessageChannel}
+    separatecontext: false
+    skipunavailable: true
+    task:
+      brand: SlackV3
+      description: Sends a message to a user, group, or channel.
+      id: dfb76202-de06-43c1-82b4-7fe51c9828d5
+      iscommand: true
+      name: Send Slack message
```
Comment (BigEasyJ) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml:202]: Before sending the Slack message we should check that slack is enabled, unless you think the check in RPR is enough.

File: Packs/CortexAttackSurfaceManagement/README.md (hunk 2 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/README.md]: Fixed in https://github.com/demisto/content/pull/34827/commits/6aebe6d769db73660d6872e1ff3233f786efce23

good catch!

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml (hunk 3 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml:143]: I would saw that a failures should be good enough (what we do for other notifications), but I am open for other options

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml (hunk 4 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml:202]: yes, we are effectively checking slack is enabled in RPR (just like for Jira and SNOW)

**Review by capanw — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@Shellyber / @ssokolovich , please push when possible

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml
[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Remediation_Path_Rules.yml:1]: fixed in [update playbook README](https://github.com/demisto/content/pull/34827/commits/0f7b362dd7dd3096d5f3eee73448a9672f4a47ec)

**Review by Shellyber — APPROVED** [review]

**Pull request merged**